.travis.yml

```diff
@@ -8,8 +8,13 @@ services:
 install:
   - pip install pipenv
   - pipenv sync
+env:
+  matrix:
```

Comment:
AFAIK `matrix` is deprecated and you should use `jobs`

Reply:
done